src/App.js

```diff
@@ -11,7 +11,18 @@ export default function App() {
   console.log({ data });
 
   // Write code here.
-  //
+
+    async function triggerfetch() {
+        const response = await fetch('https://swapi.dev/api/${dataType}/' + {dataType})
```

Comment:
You'll want to remove the curly braces from around `dataType` otherwise you're appending an object onto a string